resources/public/api/conf/1.0/application.raml

```diff
@@ -81,33 +81,29 @@ traits:
         200:
           body:
             application/json:
-              description: JSON Payload
+              description: JSON payload
```

Comment:
It should be "JSON Payload" to match the string casing in the xml payload string description.